Documentation/nvme-create-ns.txt

```diff
@@ -16,6 +16,7 @@ SYNOPSIS
 			[--anagrp-id=<anagrpid> | -a <anagrpid>]
 			[--nvmset-id=<nvmsetid> | -i <nvmsetid>]
 			[--csi=<command_set_identifier> | -y <command_set_identifier>]
+			[--lbstm=<lbstm> | -l <lbstm>]
```

Comment:
btw, it's okay just to update the *.txt files. I'll recreate the other documentation before each release.

Reply:
Good to know, I can remove the .html and .1 file changes if that'd be best?